client/lib/plans/plans-list.js

```diff
@@ -90,11 +97,13 @@ const getPlanPersonalDetails = () => ( {
 				},
 			}
 		),
-	getShortDescription: () =>
-		i18n.translate(
-			'Boost your website with a custom domain name, and remove all WordPress.com advertising. ' +
-				'Get access to high-quality email and live chat support.'
-		),
+	getShortDescription: isEligibleForPlanStepUpdates =>
+		isEligibleForPlanStepUpdates
```

Comment:
The copy switching isn't working for me. I think it's because `isEligibleForPlanStepUpdates` isn't a Boolean, but a function, namely `abtest`?

[Here is where it's called.](https://github.com/Automattic/wp-calypso/blob/45a6438d2b328436733c884c2eb69a13e622b640/client/my-sites/plan-features/index.jsx#L585)

**Before** (planStepCopyUpdates control)
<img width="1400" alt="Screen Shot 2020-02-26 at 10 15 21 am" src="https://user-images.githubusercontent.com/6458278/75296788-4e3f8580-5882-11ea-860a-615e8beefaa0.png">


**After** (planStepCopyUpdates variant)
<img width="1410" alt="Screen Shot 2020-02-26 at 10 14 40 am" src="https://user-images.githubusercontent.com/6458278/75296802-53043980-5882-11ea-97a4-6bb6cfe7ca34.png">

Reply:
> The copy switching isn't working for me.

Urgghh... thanks for spotting that 🙇 

I've fixed it in [10b0cb9](https://github.com/Automattic/wp-calypso/pull/39621/commits/10b0cb96f6aed420aaccf19aee829e0afbe5004c).